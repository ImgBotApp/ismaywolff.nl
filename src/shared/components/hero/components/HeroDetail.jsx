import styled from 'styled-components'

const HeroDetail = styled.div`
  align-items: center;
  background: var(--color-transparent-black);
  bottom: 0;
  color: var(--color-white);
  display: flex;
  left: 0;
  padding: var(--size-small);
  position: absolute;
  right: 0;
`

export default HeroDetail
