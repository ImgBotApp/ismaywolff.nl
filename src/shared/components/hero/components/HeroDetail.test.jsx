/**
 * @jest-environment node
 */

import 'jest-styled-components'
import React from 'react'
import { shallow } from 'enzyme'
import HeroDetail from './HeroDetail'

describe('<HeroDetail />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<HeroDetail />)
    expect(wrapper).toMatchStyledComponentsSnapshot()
  })
})
