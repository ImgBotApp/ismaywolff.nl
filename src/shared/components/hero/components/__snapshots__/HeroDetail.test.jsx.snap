// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HeroDetail /> renders correctly 1`] = `
.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background: linear-gradient(to bottom,rgba(0,0,0,0) 0%,var(--color-transparent-black) 100%);
  bottom: 0;
  color: var(--color-white);
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  left: 0;
  padding: var(--size-small);
  position: absolute;
  right: 0;
}

<div
  className="c0"
/>
`;
